serverless.yml

```diff
@@ -0,0 +1,46 @@
+service: aqts-ts-type-router
+
+provider:
+  name: aws
+  region: ${opt:region, 'us-west-2'}
+  stage: ${opt:stage, 'TEST'}
+  runtime: java8
```

Comment:
Just confirming we want to use java8 rather than java11 here.

Reply:
For this go around, yes java8. java11 was causing issues and appeared to require more memory/time to load